packages/textarea/src/ux-textarea.css

```diff
@@ -4,49 +4,213 @@
   box-sizing: border-box;
   flex-wrap: wrap;
   flex-direction: row;
-  align-items: center;
+  align-items: flex-start;
```

Comment:
what visual bug/styling annoyance that you encounter with this?

Reply:
On top of my head I remember that I first had issues positioning the label, especially when the textarea grow to fit its content. If I remember correctly, aligning items with `flex-start` helped me to achieve right positioning with floating labels and outline labels.